package.json

```diff
@@ -33,7 +37,9 @@
     "html-webpack-plugin": "^3.2.0",
     "husky": "^2.3.0",
     "jest": "^24.8.0",
+    "jest-mock-random": "^1.0.2",
     "prettier": "^1.17.1",
+    "ts-jest": "^24.0.2",
```

Comment:
Why did you add `ts-jest`?

Reply:
because VS-code was upset about `describe` and `it`, etc.

Reply:
Now that you pointed out the presence of `@types/jest`, I will troubleshoot!